.github/workflows/release.yml

```diff
@@ -136,6 +136,10 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Create web-dist artifact
+        working-directory: web/dist
+        run: tar czf ../../dist/web-dist.tar.gz ./*
```

Comment:
Can we change this around so the working-dir is default and we call `tar czf dist/web-dist.tar.gz ./web/dist/`? Looks clearer to me